swaScraper.py

```diff
@@ -78,7 +78,10 @@ def direct_load(args, browser):
     commandString += ("destinationAirportCode=%s&" % args.arrive)
 
     # How is the fare to be paid, USD or POINTS
-    commandString += "fareType=USD&"
+    if args.points and int(args.points) > 0:
+    	commandString += "fareType=POINTS&"
+    else:
+    	commandString += "fareType=USD&"
```

Comment:
Fix inconsistent indentation

Reply:
I'll take a look

Reply:
Didn't figure this one out so you might take a look

Reply:
Don't see any warnings, looks good to me.